osu.Game/Screens/Play/PlayerSettings/BeatmapOffsetControl.cs

```diff
@@ -183,7 +184,7 @@ private void scoreChanged(ValueChangedEvent<ScoreInfo?> score)
             if (score.NewValue == null)
                 return;
 
-            if (score.NewValue.Mods.Any(m => !m.UserPlayable))
+            if (score.NewValue.Mods.Any(m => !m.UserPlayable || m is ModRelax))
```

Comment:
It should still show with taiko relax (not yet implemented though, see https://github.com/ppy/osu/issues/7914).

Reply:
I'm not sure taiko relax should return under the "relax" name yet..

Reply:
Here's an alternative to scope this change down further if preferred:

```diff
diff --git a/osu.Game.Rulesets.Osu/Mods/OsuModRelax.cs b/osu.Game.Rulesets.Osu/Mods/OsuModRelax.cs
index 32ffb545e0..aaa7c70a8d 100644
--- a/osu.Game.Rulesets.Osu/Mods/OsuModRelax.cs
+++ b/osu.Game.Rulesets.Osu/Mods/OsuModRelax.cs
@@ -18,7 +18,7 @@
 
 namespace osu.Game.Rulesets.Osu.Mods
 {
-    public class OsuModRelax : ModRelax, IUpdatableByPlayfield, IApplicableToDrawableRuleset<OsuHitObject>, IApplicableToPlayer
+    public class OsuModRelax : ModRelax, IUpdatableByPlayfield, IApplicableToDrawableRuleset<OsuHitObject>, IApplicableToPlayer, IHasNoTimedInputs
     {
         public override LocalisableString Description => @"You don't need to click. Give your clicking/tapping fingers a break from the heat of things.";
 
diff --git a/osu.Game/Screens/Play/PlayerSettings/BeatmapOffsetControl.cs b/osu.Game/Screens/Play/PlayerSettings/BeatmapOffsetControl.cs
index 8d5459809e..b542707185 100644
--- a/osu.Game/Screens/Play/PlayerSettings/BeatmapOffsetControl.cs
+++ b/osu.Game/Screens/Play/PlayerSettings/BeatmapOffsetControl.cs
@@ -184,7 +184,7 @@ private void scoreChanged(ValueChangedEvent<ScoreInfo?> score)
             if (score.NewValue == null)
                 return;
 
-            if (score.NewValue.Mods.Any(m => !m.UserPlayable || m is ModRelax))
+            if (score.NewValue.Mods.Any(m => !m.UserPlayable || m is IHasNoTimedInputs))
                 return;
 
             var hitEvents = score.NewValue.HitEvents;

```

Reply:
I think applying above as proposed is fine.